query/transformation.go

```diff
@@ -42,7 +42,7 @@ func transformTimeseries(series api.Timeseries, transform transform, parameters
 // timerangeScale takes an api.Timerange and computes the "scale" or duration of one sample (in seconds).
 // It is useful for transformations that normalize on time (like derivative or integral).
 func timerangeScale(timerange api.Timerange) float64 {
-	return float64(timerange.Resolution)
+	return float64(timerange.Resolution())
```

Comment:
Any uses of `Timerange` fields should always expect millis. If it's necessary to convert between seconds and millis, that should be done in the context of whatever needs it (e.g. if your backend could only handle seconds, convert to seconds only when you're calling out).